templates/web/site-config.ftl

```diff
@@ -47,6 +47,7 @@
     <script src="/studio/static-assets/scripts/animator.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
     <script src="/studio/static-assets/scripts/communicator.js?version=${UIBuildId!.now?string('Mddyyyy')}" ></script>
     <script src="/studio/static-assets/libs/js-cache/cache.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
+    <script src="/studio/static-assets/libs/bootstrap/js/bootstrap.min.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
```

Comment:
jQuery ui tooltips are already in the runtime. Use that instead?

Reply:
I'm using popover not tooltips. Looks better.

Reply:
We're importing all of bootstrap to use 1 component. Let's import the component(s) we will use.

Reply:
Per our conversation this will be address on other ticket.

Reply:
Per our conversation this will be address on other ticket.